src/services/query.ts

```diff
@@ -109,8 +107,12 @@ export class QueryTermsService {
   }
 }
 
-const alphabeticallyByPrefLabel = (a: Term, b: Term) => {
+const alphabeticallyByLabel = (a: Term, b: Term) => {
   const prefLabelA = a.prefLabels[0]?.value ?? '';
+  const altLabelA = a.altLabels[0]?.value ?? '';
```

Comment:
A term could have multiple altLabels. Is it relevant to take all labels into account, not just the first one?

Reply:
I haven't found a use case that demonstated the need for sorting on all the altLabels. My suggestion is to start with just the first altLabel and see if we run into this use case at some point.